RecoLocalCalo/HcalRecProducers/python/HBHEPhase1Reconstructor_cfi.py

```diff
@@ -9,6 +9,11 @@
 hbheprereco = cms.EDProducer(
     "HBHEPhase1Reconstructor",
 
+    # Label for the HcalPulseShapeLookup object and flag
+    # indicating whether we are using it
+    channelShapesLabel = cms.string("HcalDataShapes"),
+    useChannelShapes = cms.bool(False),
+
```

Comment:
Hi, I'm not a reviewer, I just have a general question on these developments.

What is their target (if there is one) for going in production ? 2025 data-taking ?

Are these changes only needed for the HCAL offline reconstruction, or are they also supposed to be used at HLT ?

Ever since the heterogeneous version of the HCAL local reconstruction was ported to Alpaka (https://github.com/cms-sw/cmssw/pull/44910), the plugin `HBHEPhase1Reconstructor` is not used anymore in the Run-3 HLT menus (in other words, HLT is blind to this PR, as far as I understand).

Reply:
This will be useful for 2023 and 2024 re-reco as well as for the future data taking. Unfortunately, at the moment we don't have an alpaka expert in the HCAL DPG, so there is no concrete plan how to get this to work in the HLT. Of course, help from Martin Kwok (or any other alpaka expert) would be appreciated.

Reply:
Thanks for the clarification.

Notifying @kakwok, @fwyzard and @cms-sw/hlt-l2.

Reply:
I have to say that this

> for the future data taking

combined with

> there is no concrete plan how to get this to work in the HLT.

sounds to me like the HCAL DPG is not taking full ownership of the HCAL code running in production today in the HLT. I don't know the quantitative impact of these channel-dependent pulse shapes, but I would assume that we (well, HCAL specifically) should make an effort to keep HLT and offline as close as possible to each other.